### ISSUE TITLE
No List in DataRecordValue

### ISSUE THREAD

**Issue by PhilVince96** [ad1217/rs-ipfix-rw#1]
For the IANA IPFIX Entity 484 bgpSourceCommunityList you would need a List of u32. How do I model that with the DataRecordValue?
Or is the Entity 484 bgpSourceCommunityList structured data because it is a list?

Thank you! 😊

**Comment by ad1217** [comment]
That is correct: a list is structured data. I never implemented support for them because the project I wrote this for didn't use them.  That project also isn't using Rust anymore, so I haven't done much with this library since. You might want to check out https://github.com/NetGauze/NetGauze if you haven't yet; their IPFIX support seemed good last I looked.
